packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -494,7 +494,7 @@ class OverflowMenu extends Component {
     const childrenWithProps = React.Children.toArray(children).map(
       (child, index) =>
         React.cloneElement(child, {
-          closeMenu: this.closeMenu,
+          closeMenu: child.props.closeMenu || this.closeMenu,
```

Comment:
Would we need to run both functions instead of deferring to a given `child` `closeMenu`? Seems like `this.closeMenu` has some state logic that we want to run but I might be misunderstanding 👀

Reply:
the main intention is to allow a function that is different from or replaces the default `closeMenu` (without expanding the component API). if the user still wants to run the existing logic they can use the existing `onClose` prop. maybe it would be more preferable to just add a boolean prop that controls whether or not the menu will close on selection of an item?